src/main/java/org/example/adapter/in/PrintingService.java

```diff
@@ -0,0 +1,7 @@
+package org.example.adapter.in;
+
+import java.util.Scanner;
+
```

Comment:
co to jest?

Reply:
My mistake -> wanted to provide a visual interface (in this time the console Type one).Deleted it.
Resolved.